setup.py

```diff
@@ -8,7 +8,7 @@
 install_requires = [
     # pegged to also match items in `extras_require`
     'botocore>=1.34.41,<1.34.52',
-    'aiohttp>=3.7.4.post0,<4.0.0',
```

Comment:
need to update github actions (python-package.yml) to only test with aiohttp > 3.9.2 and update

Reply:
Haven't I done that already?